Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\r\n/shelf/\r\n/workspace.xml\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	(revision 44a44eba0c069fb90961f6fb24bc1bab39b99726)
+++ b/.idea/.gitignore	(date 1711306611906)
@@ -1,3 +1,5 @@
 # Default ignored files
 /shelf/
 /workspace.xml
+/.idea
+.idea
\ No newline at end of file
